holidays/countries/argentina.py

```diff
@@ -24,94 +24,260 @@
+            (FEB, 27, bicentennial_national_flag),
+            (APR, 30, arg_bridge_public_holiday),
+            (SEP, 24, bicentennial_battle_tucuman),
+            (DEC, 24, arg_bridge_public_holiday),
         ),
         2013: (
-            (
-                JAN,
-                31,
-                tr(
-                    "Bicentenario de la sesión inaugural de la Asamblea "
-                    "Nacional Constituyente del año 1813"
-                ),
-            ),
-            (FEB, 20, tr("Bicentenario de la Batalla de Salta")),
+            (JAN, 31, bicentennial_assembly_1813),
+            (FEB, 20, bicentennial_battle_salta),
+            (APR, 1, arg_bridge_public_holiday),
+            (JUN, 21, arg_bridge_public_holiday),
+        ),
+        2014: (
+            (MAY, 2, arg_bridge_public_holiday),
+            (DEC, 26, arg_bridge_public_holiday),
+        ),
+        2015: (
+            (MAR, 23, arg_bridge_public_holiday),
+            (DEC, 7, arg_bridge_public_holiday),
+        ),
+        2016: (
+            (JUL, 8, arg_bridge_public_holiday),
+            (DEC, 9, arg_bridge_public_holiday),
+        ),
+        2018: (
+            (APR, 30, arg_bridge_public_holiday),
+            (DEC, 24, arg_bridge_public_holiday),
+            (DEC, 31, arg_bridge_public_holiday),
+        ),
+        2019: (
+            (JUL, 8, arg_bridge_public_holiday),
+            (AUG, 19, arg_bridge_public_holiday),
+            (OCT, 14, arg_bridge_public_holiday),
+        ),
+        2020: (
+            (MAR, 23, arg_bridge_public_holiday),
+            (JUL, 10, arg_bridge_public_holiday),
+            (DEC, 7, arg_bridge_public_holiday),
+        ),
+        2021: (
+            (MAY, 24, arg_bridge_public_holiday),
+            (OCT, 8, arg_bridge_public_holiday),
+            (NOV, 22, arg_bridge_public_holiday),
+        ),
+        2022: (
+            (MAY, 18, national_census_2022),
+            (OCT, 7, arg_bridge_public_holiday),
+            (NOV, 21, arg_bridge_public_holiday),
+            (DEC, 9, arg_bridge_public_holiday),
+        ),
+        2023: (
+            (MAY, 26, arg_bridge_public_holiday),
+            (JUN, 19, arg_bridge_public_holiday),
+            (OCT, 13, arg_bridge_public_holiday),
         ),
     }
-    default_language = "es"
 
     def _populate(self, year):
-        # The "movable holidays" whose dates coincide with Tuesdays and
-        # Wednesdays will be moved to the previous Monday. Those that
-        # coincide with Thursday, Friday, Saturday and Sunday will be
-        # moved to the following Monday.
+        """
+        !!! Movable Holidays Law !!!
+
+        - Decreto 1584/2010: 2010-11-03
+            - AUG 17, OCT 12, NOV 20 Holidays will always be on MON
+        - Decreto 52/2017: 2017-01-23 (Reconfirmed in Ley 27399)
+            - If TUE/WED ← observed on previous MON
+            - If THUR/FRI ← observed on next MON
```

Comment:
Let's keep it simple in terms of unusual/irregular/unnecessary characters.
```suggestion
            - If THU/FRI - observed on next MON
```

Reply:
Fixed.